src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -454,11 +454,16 @@ private function buildTableRows($rows)
      * @param int   $line
      *
      * @return array
+     *
+     * @throws InvalidArgumentException
      */
     private function fillNextRows(array $rows, $line)
     {
         $unmergedRows = array();
         foreach ($rows[$line] as $column => $cell) {
+            if (null !== $cell && !is_string($cell) && !$cell instanceof TableCell && !is_scalar($cell)) {
```

Comment:
What about dropping the `is_string()` check? It's already covered by the `is_scalar()` test.

Reply:
ping @Simperfit

Reply:
done